viral/wheel_freeze_movement.py

```
@@ -0,0 +1,187 @@
+
+    event_boundaries_post = merge_close_events(
+        process_motion_energy(motion_energy_post), 30
+    )
+
+    pre_freeze_path, post_freeze_path = extract_freeze_paths(row, date, mouse_name)
+
+    for name, event_boundaries, video_path in zip(
+        ["pre", "post"],
+        [event_boundaries_pre, event_boundaries_post],
+        [pre_freeze_path, post_freeze_path],
+    ):
```

Ruff (0.16.0)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)